kogito-business-rules-springboot/README.md

```diff
@@ -58,8 +92,25 @@ To make use of this application it is as simple as putting a sending request to
 
 ```
 
-Complete curl command can be found below:
+Complete curl command can be found below for adults:
 
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"person" : {"name" : "john", "age" : 20}}' http://localhost:8080/persons
 ```
+
+After the Curl command you should see a similar console log
+
+<p align="center"><img src="docs/images/consoleLog.png"></p>
+
+
+Complete curl command can be found below for children:
+
+```
+curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"person" : {"name" : "john", "age" : 5}}' http://localhost:8080/persons
+```
+
+After the Curl command you should see a similar console log
+
+<p align="center"><img src="docs/images/consoleLog2.png"></p>
```

Comment:
better to paste the output instead of a screenshot

Reply:
changed @evacchi